### PR TITLE
Restructure "Getting started with the web" overview page

### PR DESCRIPTION
**Pull request by aasthavar**
#### Summary
Add NextMenu to point to "Installing basic software" which is next page.

#### Motivation
Was reading the docs, couldn't find the way to move to next page - unless I directly clicked the next page from the bunch of pages listed in the left.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
This is a bit complicated!

The Learning Area is supposed to be structured as a collection of _modules_, where each module has:

- an overview page
- a collection of articles designed to be read in a sequence

Example overview page: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps
Example article page: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_went_wrong

The article pages have these next/previous/up navigation buttons at the start and end of the page. But the overview page does not have these: instead it has a "Guides" section that lists all the articles in order, like https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps#guides.

The page you've edited: https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web wants to be an overview page, but doesn't list the articles in the same way, even though the module does consist of a sequence of articles, like the other modules. Perhaps that's why you found it hard to find the first page. So one thing we should do is rework this page so it has a "Guides" section that more clearly signposts the articles, as the other overview pages do.

As for including "Next" nav on this page: as mentioned above we don't do this for the other overview pages, like https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps. But perhaps we should? But if so, we should use the `{{Next}}` macro, not `{{NextMenu}}` - as it is, in your PR we have a link to the overview page on the overview page, which is weird.

Also if we do want to add a `{{Next}}` nav button to overview pages, we should do it to all of them.

I hope that makes sense! If you want to take on any of this work please let me know.

**Comment by aasthavar** [comment]
Hello @wbamberg. Thank you for explaining with crystal clarity. 
I think reworking the page to have Guides section can help clear confusion that I had, as well as keep the format consistent across all overview pages.
It would be really great to add {{Next}} on all the overview pages for a smooth navigation purpose.
I would love to work on this part.

**Comment by aasthavar** [comment]
Started with the first part - restructuring the page. I have added a section called Guides, please let me know if you think I can do it differently.
If it's okay - should I try adding {{Next}} in the top part of the page?

File: files/en-us/learn/getting_started_with_the_web/index.md [hunk 1]
```diff
@@ -20,39 +20,33 @@ _Getting started with the Web_ is a concise series introducing you to the practi
...
-
-### HTML basics
-
-Hypertext Markup Language (HTML) is the code that you use to structure your web content and give it meaning and purpose. For example, is my content a set of paragraphs or a list of bullet points? Do I have images inserted on my page? Do I have a data table? Without overwhelming you, [HTML basics](/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics) provide enough information to make you familiar with HTML.
+By working through the articles listed below,  you will go from nothing to getting your first webpage online. Let's begin our journey!
 
-### CSS basics
+- Installing basic software
+  - : When it comes to tools for building a website, there's a lot to pick from. If you're just starting, you might be confused by the array of code editors, frameworks, and testing tools out there. In [Installing basic software](/en-US/docs/Learn/Getting_started_with_the_web/Installing_basic_software), we will show you step-by-step how to install the software you need to begin some basic web development.
```
Comment (wbamberg) [inline files/en-us/learn/getting_started_with_the_web/index.md:28]:
```suggestion
- [Installing basic software](/en-US/docs/Learn/Getting_started_with_the_web/Installing_basic_software)
  - : When it comes to tools for building a website, there's a lot to pick from. If you're just starting, you might be confused by the array of code editors, frameworks, and testing tools out there. In this article, we will show you step-by-step how to install the software you need to begin some basic web development.
```

File: files/en-us/learn/getting_started_with_the_web/index.md [hunk 2]
```diff
@@ -20,39 +20,33 @@ _Getting started with the Web_ is a concise series introducing you to the practi
...
 
-Cascading Stylesheets (CSS) is the code that you use to style your website. For example, do you want the text to be black or red? Where should content be drawn on the screen? What background images and colors should be used to decorate your website? [CSS basics](/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics) take you through what you need to get started.
+- What will your website look like?
+  - : Before you start writing the code for your website, you should plan it first. What information are you showcasing? What fonts and colors are you using? [What will your website look like?](/en-US/docs/Learn/Getting_started_with_the_web/What_will_your_website_look_like) We outline a simple method you can follow to plan out your site's content and design.
```
Comment (wbamberg) [inline files/en-us/learn/getting_started_with_the_web/index.md:31]:
```suggestion
- [What will your website look like?](/en-US/docs/Learn/Getting_started_with_the_web/What_will_your_website_look_like)
  - : Before you start writing the code for your website, you should plan it first. What information are you showcasing? What fonts and colors are you using? What will your website look like? We outline a simple method you can follow to plan out your site's content and design.
```

File: files/en-us/learn/getting_started_with_the_web/index.md [hunk 3]
```diff
@@ -20,39 +20,33 @@ _Getting started with the Web_ is a concise series introducing you to the practi
...
 
-### JavaScript basics
+- Dealing with files
+  - : A website consists of many files: text content, code, stylesheets, media content, and so on. When you're building a website, you need to assemble these files into a sensible structure and make sure they can talk to one another. [Dealing with files](/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files) explains how to set up a sensible file structure for your website and what issues you should be aware of.
```
Comment (wbamberg) [inline files/en-us/learn/getting_started_with_the_web/index.md:34]:
```suggestion
- [Dealing with files](/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files)
  - : A website consists of many files: text content, code, stylesheets, media content, and so on. When you're building a website, you need to assemble these files into a sensible structure and make sure they can talk to one another. This article explains how to set up a sensible file structure for your website and what issues you should be aware of.
```

File: files/en-us/learn/getting_started_with_the_web/index.md [hunk 4]
```diff
@@ -20,39 +20,33 @@ _Getting started with the Web_ is a concise series introducing you to the practi
...
 
-JavaScript is the programming language that you use to add interactive features to your website. Some examples could be games, things that happen when buttons are pressed or data is entered in forms, dynamic styling effects, animation, and much more. [JavaScript basics](/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics) give you an idea of what is possible with this exciting language, and how to get started.
+- HTML basics
+  - : Hypertext Markup Language (HTML) is the code that you use to structure your web content and give it meaning and purpose. For example, is my content a set of paragraphs or a list of bullet points? Do I have images inserted on my page? Do I have a data table? Without overwhelming you, [HTML basics](/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics) provide enough information to make you familiar with HTML.
```
Comment (wbamberg) [inline files/en-us/learn/getting_started_with_the_web/index.md:37]:
```suggestion
- [HTML basics](/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics)
  - : Hypertext Markup Language (HTML) is the code that you use to structure your web content and give it meaning and purpose. For example, is my content a set of paragraphs or a list of bullet points? Do I have images inserted on my page? Do I have a data table? In this article provides you'll get familiar with HTML.
```

File: files/en-us/learn/getting_started_with_the_web/index.md [hunk 5]
```diff
@@ -20,39 +20,33 @@ _Getting started with the Web_ is a concise series introducing you to the practi
...
 
-### Publishing your website
+- CSS basics
+  - : Cascading Stylesheets (CSS) is the code that you use to style your website. For example, do you want the text to be black or red? Where should content be drawn on the screen? What background images and colors should be used to decorate your website? [CSS basics](/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics) take you through what you need to get started.
```
Comment (wbamberg) [inline files/en-us/learn/getting_started_with_the_web/index.md:40]:
```suggestion
- [CSS basics](/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics)
  - : Cascading Stylesheets (CSS) is the code that you use to style your website. For example, do you want the text to be black or red? Where should content be drawn on the screen? What background images and colors should be used to decorate your website? This article takes you through what you need to get started.
```

File: files/en-us/learn/getting_started_with_the_web/index.md [hunk 6]
```diff
@@ -20,39 +20,33 @@ _Getting started with the Web_ is a concise series introducing you to the practi
...
 
-Once you have finished writing the code and organizing the files that make up your website, you need to put it all online so people can find it. [Publishing your sample code](/en-US/docs/Learn/Getting_started_with_the_web/Publishing_your_website) describes how to get your simple sample code online with minimum effort.
+- JavaScript basics
+  - : JavaScript is the programming language that you use to add interactive features to your website. Some examples could be games, things that happen when buttons are pressed or data is entered in forms, dynamic styling effects, animation, and much more. [JavaScript basics](/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics) give you an idea of what is possible with this exciting language, and how to get started.
```
Comment (wbamberg) [inline files/en-us/learn/getting_started_with_the_web/index.md:43]:
```suggestion
- [JavaScript basics](/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics)
  - : JavaScript is the programming language that you use to add interactive features to your website. Some examples could be games, things that happen when buttons are pressed or data is entered in forms, dynamic styling effects, animation, and much more. In this article we'll give you an idea of what is possible with this exciting language, and how to get started.
```

File: files/en-us/learn/getting_started_with_the_web/index.md [hunk 7]
```diff
@@ -20,39 +20,33 @@ _Getting started with the Web_ is a concise series introducing you to the practi
...
 
-### How the web works
+- Publishing your website
+  - : Once you have finished writing the code and organizing the files that make up your website, you need to put it all online so people can find it. [Publishing your sample code](/en-US/docs/Learn/Getting_started_with_the_web/Publishing_your_website) describes how to get your simple sample code online with minimum effort.
```
Comment (wbamberg) [inline files/en-us/learn/getting_started_with_the_web/index.md:46]:
```suggestion
- [Publishing your sample code](/en-US/docs/Learn/Getting_started_with_the_web/Publishing_your_website) 
  - : Once you have finished writing the code and organizing the files that make up your website, you need to put it all online so people can find it. This article describes how to get your simple sample code online with minimum effort.
```

File: files/en-us/learn/getting_started_with_the_web/index.md [hunk 8]
```diff
@@ -20,39 +20,33 @@ _Getting started with the Web_ is a concise series introducing you to the practi
...
 
-When you access your favorite website, a lot of complicated things happen in the background that you may not know about. [How the web works](/en-US/docs/Learn/Getting_started_with_the_web/How_the_Web_works) outlines what happens when you view a webpage on your computer.
+- How the web works
+  - : When you access your favorite website, a lot of complicated things happen in the background that you may not know about. [How the web works](/en-US/docs/Learn/Getting_started_with_the_web/How_the_Web_works) outlines what happens when you view a webpage on your computer.
```
Comment (wbamberg) [inline files/en-us/learn/getting_started_with_the_web/index.md:49]:
```suggestion
- [How the web works](/en-US/docs/Learn/Getting_started_with_the_web/How_the_Web_works) 
  - : When you access your favorite website, a lot of complicated things happen in the background that you may not know about. Here we will outline what happens when you view a webpage on your computer.
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR, @aasthavar !

If you look at, say, https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps, you'll see that the name of the article in the list is made into a link, like this:

******

<dl>
<dt><a href="https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_is_JavaScript">What is JavaScript?</a></dt>
<dd>Welcome to the MDN beginner's JavaScript course! In this first article...</dd>
</dl>

***********

I think we should do the same here, instead of embedding the link in the paragraph. That also means we should adjust some of the text.

I've added some suggestion for how we could do this. It is a bit repetitive in places ("This article..."), but I think it should be OK.

**Comment by aasthavar** [comment]
Thank you for your suggestions! I have tried to modify the same.

**Review by wbamberg — APPROVED** [review]
👍 looks great, thank you, and congratulations on getting your first mdn/content PR merged!

**Comment by aasthavar** [comment]
Thank you :)

**Pull request merged**